### DISCUSSION TITLE
Standalone Executable for Windows

### DISCUSSION THREAD

**Discussion by koochack** [abhinavsingh/proxy.py#1471] [Q&A] [answered]
Hello, I was wondering if there's a way to build a standalone, single executable file to remove any dependencies to Python installations/packages or path variables? I'd like to use proxy.py outside of Python environment and make a portable .exe file.

**Comment by abhinavsingh** [comment, marked as answer]
You can use `pyinstaller`.   We have successfully packaged `proxy.py` as a standalone distributable desktop installer for Mac (signed), Windows and Linux.

**Comment by koochack** [comment]
Thank you for quick response. I was able to create the executable with
pyinstaller 🥳🙏🏻 it was very difficult and tricky though!
Where can I download the distributable installer? It's probably better than my version since even though it runs fine when executed from Windows explorer, I am having issues running it as a plug-in of Nekoray. I want to give your executable version a try

**Reply by abhinavsingh** [reply]
1) Yes bundling a library within `pyinstaller` can be tricky, specially considering that `proxy.py` heavily uses `multiprocessing`
2) Sorry, currently there are no public `pyinstaller` executable availabe as part of the `proxy.py` project
